### PR TITLE
Fix a few type errors

### PR DESCRIPTION
**Pull request by mathiasvr**
**What is the purpose of this pull request? (put an "X" next to item)**

[ ] Documentation update
[ ] Bug fix
[ ] New feature
[x] Other, please explain:

**What changes did you make? (Give an overview)**
Fixes type errors, mainly according to new Electron version.

### CODE REVIEW

File: src/main/windows/webtorrent.js [hunk 1]
```diff
@@ -26,7 +25,8 @@ function init () {
     title: 'webtorrent-hidden-window',
     useContentSize: true,
     webPreferences: {
-      nodeIntegration: true
+      nodeIntegration: true,
+      backgroundThrottling: false // do not throttle animations/timers when page is background
```
Comment (mathiasvr) [inline src/main/windows/webtorrent.js]: I'm not sure this was in effect before, do we actually want to disable background throttling?

Comment (hicom150) [reply src/main/windows/webtorrent.js]: Given that it was not in inside `webPreferences` I assume it was in it's default state: `true`. So I wouldn't change it here 😅

File: src/main/windows/main.js [hunk 2]
```diff
@@ -44,7 +43,8 @@ function init (state, options) {
     useContentSize: true, // Specify web page size without OS chrome
     width: initialBounds.width,
     webPreferences: {
-      nodeIntegration: true
+      nodeIntegration: true,
+      backgroundThrottling: false // do not throttle animations/timers when page is background
```
Comment (hicom150) [inline src/main/windows/main.js]: Given that it was not in inside `webPreferences` I assume it was in it's default state: `true`. So I wouldn't change it here 😅

File: src/renderer/pages/player-page.js [hunk 3]
```diff
@@ -497,7 +497,7 @@ function renderPlayerControls (state) {
       <div
         key='scrub-bar'
         className='scrub-bar'
-        draggable='true'
+        draggable
```
Comment (hicom150) [inline src/renderer/pages/player-page.js]: I not sure but I think that we should leave it as it was according to [draggable documentation](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/draggable)

**Comment by hicom150** [comment]
Great work @mathiasvr 👍
I point just a pair of things to check 😉

File: src/renderer/pages/player-page.js (hunk 3 above)
Comment (mathiasvr) [reply src/renderer/pages/player-page.js]: The jsx draggable type is boolean (not string), and this is shorthand for `draggable={true}`. I believe that should be correct according to the docs.

**Comment by mathiasvr** [comment]
@hicom150 Thanks for the review, I have updated the PR to remove the `backgroundThrottling: false` setting.

**Comment by feross** [comment]
I actually think we want `backgroundThrottling: false` so our timers and intervals aren't throttled when the app is backgrounded.

**Pull request merged**